src/pages/Home/Home.styled.jsx

```diff
@@ -75,19 +82,10 @@ export const ManImage = styled.img`
     align-self: center;
     margin-top: 30%;
     height: 68vh;
-  }
-
-  @media (min-width: 960px) {
-    margin-top: 12%;
-  }
+    margin: 0;
```

Comment:
this overrides `margin-top` on line 83, can you verify if this is correct? If one rule is not needed please erase it